examples/lighting-app/qpg/src/LightingManager.cpp

```diff
@@ -21,12 +21,138 @@
 #include "qvCHIP.h"
 #include <support/logging/CHIPLogging.h>
 
+#include <math.h>
+
+// initialization values for Blue in XY color space
+#define BLUE_XY                                                                                                                    \
+    {                                                                                                                              \
+        9830, 3932                                                                                                                 \
+    }
+
+// initialization values for Blue in HSV color space
+#define BLUE_HSV                                                                                                                   \
+    {                                                                                                                              \
+        240, 100, 255                                                                                                              \
+    }
+
+// default initialization value for the light level after start
+#define DEFAULT_LEVEL (64)
+
 LightingManager LightingManager::sLight;
 
+RgbColor_t LightingManager::HsvToRgb(HsvColor_t hsv)
```

Comment:
Is this conversion device-specific?  Or would it make sense to have a utility for this in the color control cluster bits?

Reply:
It is a generic transformation that would benefit all platforms - but I'm unsure which would be a better place for it in that context.